hs/t/y/Fix_Shadowed.txt

```diff
@@ -0,0 +1,6 @@
+
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+Checked 1 theorems; No failures!
```

Comment:
This is incorrect.

[`hs-test` failed](https://app.circleci.com/pipelines/github/reach-sh/reach-lang/7346/workflows/fbc84698-6468-421e-802a-620e8cab7ff5/jobs/133922).

```
Test output was different from '/build/t/y/./Fix_Shadowed.txt'. Output of ["diff","-u","/build/t/y/./Fix_Shadowed.txt","/tmp/Fix_Shadowed216-96.actual"]:
--- /build/t/y/./Fix_Shadowed.txt
+++ /tmp/Fix_Shadowed216-96.actual
@@ -1,6 +1,13 @@
-
 Verifying knowledge assertions
 Verifying for generic connector
   Verifying when ALL participants are honest
   Verifying when NO participants are honest
 Checked 1 theorems; No failures!
+warning[RW0004]: There are no publications in the application.
+
+  ./Fix_Shadowed.rsh:6:30:application
+
+  6| export const main = Reach.App(
+
+For further explanation of this warning, see: https://docs.reach.sh/rsh/errors/#RW0004
+
```

You should be able to use [the new version of `scripts/gen-test-output.sh`](https://github.com/reach-sh/reach-lang/pull/1095/files#diff-bfd62d82e917b4bb32e86f0a3d873ce7ca29b3bde6485b7c01f33b3dd6f6c784) to get the correct output.

You can `git rebase` or copy and paste the script to use it.

Don't forget to do `brew install gnu-sed`!

You can direct message me on Slack for additional assistance if you need it.